Practice/case.py

```diff
@@ -0,0 +1,86 @@
+"""case"""
+import re
+
+
+def get_input():
+    """
+    Takes the input from the user"""
+    print('Please enter a word to be formatted:')
+    return input()
+
+
+def test_format(word):
```

Comment:
I think this function could have a better name. What does it "test"? What is a "format"? The problem uses the language "case" to describe which case a word is in. Maybe `find_case`? Up to you; I'm sure you could come up with a better name too.